src/App.js

```diff
@@ -1,11 +1,32 @@
 /* eslint-disable react/prefer-stateless-function */
 import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import Calculator from './component/Calculator';
+import Navfun from './component/navbar';
+import Homefun from './component/Home';
+import Quote from './component/Quote';
 
```

Comment:
The general layout of some of your pages **do not match** the provided [wireframe](https://github.com/microverseinc/curriculum-react-redux/tree/main/math-magicians/images). Kindly ensure to improve upon the following:
- [ ] The Home page should have some **paragraph text** after the heading _Welcome to our page_.
- [ ] The text content on the Quote page should be **vertically centered**.

**Your Pages:**
![image](https://user-images.githubusercontent.com/12745474/141191034-b2d3596c-6722-488c-bf33-e631170bb2d6.png)

![image](https://user-images.githubusercontent.com/12745474/141191067-cf620778-e577-4c5f-ade9-ca3a68d22454.png)


**The wireframe pages:**
![image](https://user-images.githubusercontent.com/12745474/141191446-6a4d7184-1fb6-4a29-9423-29d2d0236649.png)

![image](https://user-images.githubusercontent.com/12745474/141191558-6c4e3c81-046b-4364-b097-3b2482d435b4.png)